samples/XCT.Sample.Tizen/Xamarin.CommunityToolkit.Sample.Tizen.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Xamarin.CommunityToolkit\Xamarin.CommunityToolkit.csproj" />
+    <ProjectReference Include="..\..\src\CommunityToolkit\Xamarin.CommunityToolkit\Xamarin.CommunityToolkit.csproj" />
     <ProjectReference Include="..\XCT.Sample\Xamarin.CommunityToolkit.Sample.csproj" />
```

Comment:
Hmmm, this guy doesn't seem to be right too...

Reply:
This address is the same as the shared sample project, as you can see [here](https://github.com/xamarin/XamarinCommunityToolkit/blob/main/samples/XCT.Sample/Xamarin.CommunityToolkit.Sample.csproj#L24). If you run the old address the Tizen project will complain.